diff --git a/arch/arm/boot/dts/qcom/sdm660-regulator.dtsi b/arch/arm/boot/dts/qcom/sdm660-regulator.dtsi
index b046f106d95c..9169671fbda3 100644
--- a/arch/arm/boot/dts/qcom/sdm660-regulator.dtsi
+++ b/arch/arm/boot/dts/qcom/sdm660-regulator.dtsi
@@ -597,11 +597,11 @@
 				qcom,cpr-corner-fmax-map = <1 2 3 4 5 6>;
 
 				qcom,cpr-voltage-ceiling =
-					<585000  645000  725000  790000
-					 870000  925000 1070000>;
+					<545000  605000  685000  750000
+					 830000  885000 1030000>;
 				qcom,cpr-voltage-floor =
-					<504000  504000  596000  652000
-					 712000  744000 1070000>;
+					<464000  464000  556000  612000
+					 672000  704000 1030000>;
 
 				qcom,mem-acc-voltage = <1 1 1 2 2 2 2>;
 				qcom,system-voltage =
@@ -747,12 +747,12 @@
 					<2 3 4 5 8>;
 
 				qcom,cpr-voltage-ceiling =
-					< 724000  724000  724000  788000  868000
-					 1068000 1068000 1068000>;
+					< 684000  684000  684000  748000  828000
+					 1028000 1028000 1028000>;
 
 				qcom,cpr-voltage-floor =
-					<588000  588000  596000  652000  712000
-					 744000  784000  844000>;
+					<548000  548000  556000  612000  672000
+					 704000  744000  804000>;
 
 				qcom,corner-frequencies =
 					/* Speed bin 0 */
@@ -898,12 +898,12 @@
 					<2 3 4 6 7>;
 
 				qcom,cpr-voltage-ceiling =
-					<724000  724000  788000  868000
-					 988000  988000 1068000>;
+					<684000  684000  748000  828000
+					 948000  948000 1028000>;
 
 				qcom,cpr-voltage-floor =
-					<588000  596000  652000  712000
-					 744000  784000  844000>;
+					<548000  556000  612000  672000
+					 704000  744000  804000>;
 
 				qcom,corner-frequencies =
 					/* Speed bin 0 */
